libsolidity/analysis/DeclarationContainer.cpp

```diff
@@ -50,7 +50,8 @@ Declaration const* DeclarationContainer::conflictingDeclaration(
 	if (
 		dynamic_cast<FunctionDefinition const*>(&_declaration) ||
 		dynamic_cast<EventDefinition const*>(&_declaration) ||
-		dynamic_cast<MagicVariableDeclaration const*>(&_declaration)
+		dynamic_cast<MagicVariableDeclaration const*>(&_declaration) ||
```

Comment:
Isn't this the list of symbols that can be overloaded? I'm not sure `UserDefinedValueType` belongs here.